src/pybind/cephfs.py

```diff
@@ -82,6 +83,12 @@ class cephfs_statvfs(Structure):
                 ("f_flag", c_uint),
                 ("f_namemax", c_uint)]
 
+class cephfs_dirent(Structure):
+    _fields_ = [("d_ino", c_long),
+                ("d_off", c_ulong),
```

Comment:
I don't think a python consumer will ever be able to do anything with d_off and d_reclen, we should not pass them up.